application/src/main/kotlin/application/StubLoaderEngine.kt

```diff
@@ -24,7 +29,7 @@ class StubLoaderEngine {
 
         return when {
             dataDirs.isNotEmpty() -> {
-                loadContractStubsFromFiles(contractPathDataList, dataDirs, specmaticConfig)
+                loadContractStubsFromFiles(contractPathDataList, dataDirs, specmaticConfig, strictMode)
```

Comment:
is Strict mode not part of SpecmaticConfig? if yes, then no need to add another param right? Please correct me if I am wrong.